src/main/java/com/streamforge/realm/twitch/oauth/camel/TwitchAuthRoute.java

```diff
@@ -76,8 +80,8 @@ public void configure() {
                         })
                         .choice()
                             .when(body().isNotNull())
-                                //todo REST API CALL HERE
-                                //the value could be retrieved as simple("${body}")
+                                .process(request -> request.getIn().setBody(saveTokenRequest(request.getIn().getBody().toString())))
```

Comment:
saveToken(simple(${body}))

Reply:
request -> ..=> its ok if just above its (process (request -> ..)) too
if not change to exchange pls